slitlessutils/core/wfss/data/wfsscollection.py

```diff
@@ -305,6 +310,36 @@ def __setitem__(self, k, v):
 
         super().__setitem__(k, v)
 
+    def as_table(self):
+        """
+        Method to output contents of a WFSSCollection as an `astropy.Table`
+
+        Parameters
+        ----------
+        None.
+
+        Returns
+        -------
+        tab : `astropy.tables.Table`
+            The astropy table
+        """
+
+        # a dict to hold the results as a start
+        cols = {'filename': [], 'visit': [], 'orbit': [], 'disperser': [], 'blocking': []}
+
+        # iterate over each datum and update cols dict
+        for dataset, data in self.items():
+            cols['filename'].append(dataset)
+            cols['visit'].append(data.visit)
+            cols['orbit'].append(data.orbit)
+            cols['disperser'].append(data.disperser.name)
+            cols['blocking'].append(data.disperser.blocking)
+
+        # repackage as a Table object
+        tab = Table(cols)
+
+        return tab
```

Comment:
```suggestion
        return Table(cols)
```
No need to set as a variable here first since it isn't used before returning.